minetest/minetest
# Bigfoot and footstep at feet

- Objects can make footstep sounds. They currently do this at their origin pos. This PR adds the lower y end of the collision box to the pos, so that it's at the feet.
- There's also a new luaentity object called "soundstuff:bigfoot" for testing.
- Does it resolve any reported issue? No.
- This can be considered a bugfix, as this behaviour makes more sense.

## To do

This PR is a Ready for Review.

Don't squash.
Review the commits separately.

## How to test

* Place a walkway of `soundsstuff:footstep` nodes in x direction.
* Stand in the middle of the walkway. There, issue `/spawn_bigfoot`.
* Fly with your head to the middle and to the base of the object. Compare the loudness. Expected: It's louder at the bottom.
* Try the same with master by removing the last commit via git-checkout.
